### PR TITLE
EPMRPP-62669 - add tooltip in case analyzer is not running

### PR DESCRIPTION
**Pull request by chivekrodis**

### CODE REVIEW

File: app/src/components/inputs/inputRadio/inputRadio.jsx [hunk 1]
```diff
@@ -92,9 +96,6 @@ InputRadio.defaultProps = {
   onChange: () => {},
   onFocus: () => {},
   onBlur: () => {},
-  inputClassNames: {
-    togglerClassName: '',
-    childrenClassName: '',
-  },
   title: '',
+  mode: '',
```
Comment (AmsterGet) [inline app/src/components/inputs/inputRadio/inputRadio.jsx:100]: May be specify a meaningful default value, f.e. `mode: default`?

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx [hunk 2]
```diff
@@ -50,6 +51,9 @@ const MakeDecision = ({ data }) => {
   const dispatch = useDispatch();
   const activeProject = useSelector(activeProjectSelector);
   const historyItems = useSelector(historyItemsSelector);
+  const isAnalyzerAvailable = useSelector(analyzerExtensionsSelector).find(
```
Comment (AmsterGet) [inline app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx:54]: Why not just check `analyzerExtensions` length?

Comment (chivekrodis) [reply app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx:54]: How can I be sure that an existing extension is a analyzer?

File: app/src/components/inputs/inputRadio/inputRadio.jsx (hunk 1 above)
Comment (chivekrodis) [reply app/src/components/inputs/inputRadio/inputRadio.jsx:100]: If I get you right, we should wrap old styles with `mode-default` class?

File: app/src/components/inputs/inputRadio/inputRadio.jsx [hunk 3]
```diff
@@ -75,11 +82,8 @@ InputRadio.propTypes = {
   onChange: PropTypes.func,
   onFocus: PropTypes.func,
   onBlur: PropTypes.func,
-  inputClassNames: PropTypes.shape({
-    togglerClassName: PropTypes.string,
-    childrenClassName: PropTypes.string,
-  }),
   title: PropTypes.string,
+  mode: PropTypes.string,
```
Comment (AmsterGet) [inline app/src/components/inputs/inputRadio/inputRadio.jsx:86]: You may specify: `mode: PropTypes.oneOf(['default', 'dark'])`.

File: app/src/components/inputs/inputRadio/inputRadio.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/components/inputs/inputRadio/inputRadio.jsx]: 'default' mode value may be used in prop definitions just to specify the way how the mode should look like, without any special styles. But separation some specific, not common styles with `mode-default` sounds good too.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**